src/pages/iou/request/step/StepScreenWrapper.tsx

```diff
@@ -25,6 +25,9 @@ type StepScreenWrapperProps = {
     /** Whether or not to display not found page */
     shouldShowNotFoundPage?: boolean;
 
+    /** Whether to show offline indicator */
+    shouldShowOfflineIndicator?: boolean;
```

Comment:
StepScreenWrapper renders ScreenWrapper which shows OfflineInidicator at bottom for small screen but our page would already show that user is offline so showing an indicator at bottom would feel weird.
https://github.com/Expensify/App/blob/9627f5f8da6542b42a7f2eb9125dc82eea083740/src/components/ScreenWrapper.tsx#L123
https://github.com/Expensify/App/blob/9627f5f8da6542b42a7f2eb9125dc82eea083740/src/components/ScreenWrapper.tsx#L288

Basically, to avoid this:
![image](https://github.com/user-attachments/assets/39a795c0-3e62-4ccf-ba05-bc4d80711e2a)
 
Another approach I tried was to add below code to IOURequestStepCategory but then the header will not render because it comes from StepScreenWrapper so the current approach seemed better
```js
if (policyCategories === undefined && isOffline) {
  // a children is mandatory for FullPageOfflineBlockingView
  return <FullPageOfflineBlockingView>{null}</FullPageOfflineBlockingView>
}
```